### PR TITLE
Hoang feature

### PR DESCRIPTION
**Pull request by hoangvu518**
Hello lead. Please review my code for this feature

### CODE REVIEW

File: ConsoleApp/ConsoleApp/Program.cs [hunk 1]
```diff
@@ -7,29 +7,8 @@ class Program
...
-            Console.WriteLine("Hello World 2!");
-            Console.WriteLine("Hello World 2!");
-            Console.WriteLine("Hello World 2!");
-            Console.WriteLine("Hello World 2!");
-            Console.WriteLine("Hello World 2!");
-            Console.WriteLine("Hello World 2!");
-            Console.WriteLine("Hello World 2!");
-            Console.WriteLine("Hello World 2!");
+            //refactored the code.\
+            //hahahaha
```
Comment (minhhieutle) [inline ConsoleApp/ConsoleApp/Program.cs:11]: no laughing allowed

**Review by minhhieutle — CHANGES_REQUESTED** [review]
no approval. see my comments

File: ConsoleApp/ConsoleApp/Program.cs (hunk 1 above)
Comment (hoangvu518) [reply ConsoleApp/ConsoleApp/Program.cs:11]: I'm sorry. Changing code now

**Review by minhhieutle — APPROVED** [review]
approved

**Pull request merged**